### DISCUSSION TITLE
Trying to install logging on OKD 4.9 (and forward logs to an external ELK)

### DISCUSSION THREAD

**Discussion by var-log-skynet** [okd-project/okd#1141] [General]
I'm following the documentation on how to install logging. I have a developer account at RedHat, I've downloaded the pull secret (https://console.redhat.com/openshift/install/pull-secret) and added it to "openshift" and "openshift-marketplace" namesapces, the enabled redhat-operators in the OperatorHub CR, but I always get this: 

`Failed to pull image "registry.redhat.io/redhat/redhat-operator-index:v4.9": rpc error: code = Unknown desc = unable to retrieve auth token: invalid username/password: unauthorized: Please login to the Red Hat Registry using your Customer Portal credentials. Further instructions can be found here: https://access.redhat.com/RegistryAuthentication
`
Following that link I created a SA, added it to the same namespaces. Same error. 

I'm on bare metal, 
`NAME      VERSION                         AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.9.0-0.okd-2022-02-12-140851   True        False         19d     Cluster version is 4.9.0-0.okd-2022-02-12-140851`

What I'm I missing here?

**Comment by vrutkovs** [comment]
See https://docs.openshift.com/container-platform/4.9/support/remote_health_monitoring/opting-out-of-remote-health-reporting.html#images-update-global-pull-secret_opting-out-remote-health-reporting

**Reply by var-log-skynet** [reply]
Thank you! Works like a charm!
